tools/traffic/workers/mock_chain_client.go

```diff
@@ -0,0 +1,24 @@
+package workers
```

Comment:
Do we need this new mock impl for chain client and retrieval client below? 
Looks like we already have mock implementations

Reply:
I've removed the new mock implementations and used the existing ones.